citesphere/src/main/webapp/WEB-INF/views/auth/authorities/list.html

```diff
@@ -1,10 +1,30 @@
 <html layout:decorate="~{layouts/main}">
 <head>
+<script th:src="@{/resources/paginator/jquery.twbsPagination.min.js}"></script>
 <script th:src="@{/resources/notify/bootstrap-notify.min.js}"></script>
 <link th:href="@{/resources/notify/animate.css}" rel="stylesheet"/>
 
 <script th:inline="javascript">
 $(function() {
+  	$('#pagination-top').twbsPagination({
+        totalPages: [(${totalPages})],
+	    startPage: [(${currentPage})],
+	    prev: "«",
+	    next: "»",
+	    visiblePages: 10,
+	    initiateStartPageClick: false,
+        onPageClick: function (event, page) {
+     	    if($("#displayAuthorities").val() === 'all'){
+     	    	 console.log("all")
+     	    	 window.location.href = [[@{/auth/authority/list}]] + "?" + "page=" + page;
+    	    } else if($("#displayAuthorities").val() === 'userSpecific'){
+    	    	window.location.href = [[@{/auth/authority/user/list}]] + "?" + "page=" + page;
+    	    } else {
+    	    	var zoteroGroupId = $("#displayAuthorities").val();
+    	    	window.location.href = [[@{/auth/authority/}]] + zoteroGroupId + "/list" + "?" + "page=" + page;
+    	    }
```

Comment:
this looks like the same code as below in the `$("#displayAuthorities").change` function. Should be DRY.